tests/Aspire.Workload.Tests/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <IsWorkloadTestProject>true</IsWorkloadTestProject>
```

Comment:
Small NIT: it is sometimes helpful to add comments above properties like this to just explain what this will do. For instance, this will import workload testing props and targets.